REUSE.toml

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2023 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+version = 1
+
+[[annotations]]
+path = [
+     ".bumpversion.cfg",
+     "setup.cfg"
+]
+precedence = "override"
+SPDX-FileCopyrightText = "2017 Free Software Foundation Europe e.V. <https://fsfe.org>"
+SPDX-License-Identifier = "GPL-3.0-or-later"
```

Comment:
I am not that big of a fan of how verbose these keys are, and how they don't follow the style of the other keys. Maybe something like this would make more sense?
```toml
copyright_text = "2017 Free Software Foundation Europe e.V. <https://fsfe.org>"
license_identifier = "GPL-3.0-or-later"
```

Reply:
Or should I move this concern/comment to https://github.com/fsfe/reuse-website?